### ISSUE TITLE
Server API plugin support

### ISSUE THREAD

**Issue by ADRS55** [Acekorneya/Ark-Survival-Ascended-Server#52]
It would be great to have support for the [Server API](https://ark-server-api.com) plugin. Is there any plans to add it?

**Comment by Acekorneya** [comment]
Yeah, I would like to be able to implement it into the script. The issue is that they don't have a Linux build. Additionally, we run ASA under Proton because the ASA devs don't have a Linux build for ASA. So, it would take some work to get the API working under Linux using Proton or Wine.

**Comment by ADRS55** [comment]
I saw that Mr. Mschnitzer managed to implement the server API in his ASA docker image. I was just curious, Don't want to migrate to another image because yours is really good :) Thanks.

**Comment by Acekorneya** [comment]
Thanks for letting me know. A lot of people have been asking if I will implement it. Now that I see others have gotten it working, I might be able to implement it in my script as well. I'll give it a shot and see if I can get it working with my version of the script.

**Comment by Acekorneya** [comment]
we have API support now just do -API true -all and should use the API
